scrapers/prowlarr.py

```
@@ -285,36 +415,88 @@ async def queue_processor():
                 f"Stream processing timed out after {max_process_time} seconds. "
                 f"Processed {streams_processed} streams"
             )
+            self.metrics.record_skip("Max process time")
         except ExceptionGroup as eg:
```

Ruff

Undefined name `ExceptionGroup`. Consider specifying `requires-python = ">= 3.11"` or `tool.ruff.target-version = "py311"` in your `pyproject.toml` file.

(F821)